generator/src/main/java/com/scottlogic/deg/generator/fieldspecs/FieldSpec.java

```diff
@@ -296,4 +287,25 @@ private static boolean propertiesAreEqual(Object myProperty, Object otherPropert
             fieldSpec.typeRestrictions
         };
     }
+
+    private static class EmptyFieldSpec extends FieldSpec {
+        public EmptyFieldSpec() {
+            super(
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                FieldSpecSource.Empty);
+        }
+
+        @Override
+        public String toString() {
+            return "<empty>";
```

Comment:
Would it make more sense to just modify the toString() of Fieldspec to return "<empty>" if it is empty instead of introducing a new subclass.

Reply:
It currently returns an empty string on `master` i wanted something a bit more explicit. It also means that this 'explicit' logic doesn't need to sit in the 'main' type.

Reply:
its still in the main type unless you move it to a new class.